src/contracts/treasury/Collector.sol

```diff
@@ -47,16 +56,18 @@ contract Collector is VersionedInitializable, ICollector, ReentrancyGuard {
    */
   mapping(uint256 => Stream) private _streams;
 
-  /// @inheritdoc ICollector
-  address public constant ETH_MOCK_ADDRESS = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+  /**
+   * @notice [DEPRECATED] Use `isFundsAdmin()` to check address.
+   */
+  address internal _fundsAdmin_deprecated;
```

Comment:
I would keep the same order for state variables, first `_fundsAdmin_deprecated` then `_nextStreamId` and then `_streams` so as to keep the storage layout aligned

Reply:
yes for sure, missed that.